src/dev/impl/DevToys/Views/Tools/Converters/NumberBaseConverter/AdvancedNumberBaseConverterControl.xaml

```diff
@@ -0,0 +1,73 @@
+﻿<UserControl
+    x:Class="DevToys.Views.Tools.Converters.NumberBaseConverter.AdvancedNumberBaseConverterControl"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:DevToys.UI.Controls"
+    xmlns:DevToys="using:DevToys"
+    xmlns:converters="using:DevToys.UI.Converters"
+    mc:Ignorable="d"
+    d:DesignHeight="400"
+    d:DesignWidth="600">
+
+    <UserControl.Resources>
+        <converters:BooleanToVisibilityConverter x:Key="BooleanToVisibilityConverter"/>
+    </UserControl.Resources>
+    
+    <StackPanel Margin="0,0,0,16" Spacing="12">
+        <StackPanel.ChildrenTransitions>
+            <TransitionCollection>
+                <EntranceThemeTransition IsStaggeringEnabled="True" FromVerticalOffset="50"/>
+            </TransitionCollection>
+        </StackPanel.ChildrenTransitions>
+
+        <StackPanel Spacing="4" >
+            <controls:ExpandableSettingControl
+                x:Name="InputDictionaryValue"
+                Grid.Row="2"
+                Title="{x:Bind ViewModel.Strings.InputDictionaryLabel}">
+                <controls:ExpandableSettingControl.Icon>
+                    <FontIcon />
```

Comment:
Try the glyph `&#x0065;`

![image](https://user-images.githubusercontent.com/3747805/158031921-091490ca-cdf4-4b58-9856-eed521078fd3.png)